python/paddle/tensor/manipulation.py

```diff
@@ -548,6 +549,22 @@ def squeeze(x, axis=None, name=None):
     return layers.squeeze(x, axis, name)
 
 
+def squeeze_(x, axis=None, name=None):
+    """
+    only used in dygraph
+    """
+    if axis is None:
+        axis = []
+    elif isinstance(axis, int):
+        axis = [axis]
+    elif isinstance(axis, tuple):
+        axis = list(axis)
+
+    assert in_dygraph_mode(), "squeeze_ can't be used in static mode"
+    core.ops.squeeze2_(x, x, 'axes', axis)
```

Comment:
The second `x` in function `core.ops.squeeze_(x, x, 'axes', axis)` is not necessary.